nautobot/core/tests/test_utilities.py

```diff
@@ -100,3 +103,73 @@ def test_check_filter_for_display(self):
             self.assertEqual(
                 check_filter_for_display(device_filter_set_filters, "manufacturer", ["fake_slug"]), expected_output
             )
+
+
+class GetFilterFieldLabelTest(TestCase):
+    """
+    Validate the operation of get_filter_field_label().
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        device_ct = ContentType.objects.get_for_model(Device)
+        cls.peer_relationship = Relationship(
+            name="HA Device Peer",
+            slug="ha-device-peer",
```

Comment:
```suggestion
            slug="ha_device_peer",
```

Reply:
Copied from the existing Relationship tests but I'll change here.